### PR TITLE
add useful part in README part

### PR DESCRIPTION
**Pull request by PeterlitsZo**
(at least it is useful for me)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,33 @@ You can implement sub-commands in your CLI using `parser.NewCommand()` or go eve
...
+type Options struct {
+	Required bool
+	Validate func(args []string) error
+	Help     string
+	Default  interface{}
+}
+```
+
+You can Set `Required` to let it know if it should ask for arguments.
+Or you can set `Validata` as a lambda function to make it know while value is valid.
```
Comment (akamensky) [inline README.md:153]: Validate

File: README.md [hunk 2]
```diff
@@ -137,6 +137,33 @@ You can implement sub-commands in your CLI using `parser.NewCommand()` or go eve
...
+Or you can set `Help` for your beautiful help document.
+Or you can set `Default` will set the default value if user do not give your programme a value.
```
Comment (akamensky) [inline README.md:155]: `if user do not provide a value`

**Comment by akamensky** [comment]
@PeterlitsZo thanks, there are two corrections I put in comments. Can you please check. Other than that it is good to be merged. thanks.

**Comment by PeterlitsZo** [comment]
Thanks. It will go well as it can. It will be good if you merge it. `^_^`

**Comment by PeterlitsZo** [comment]
(I am green in how to merge a pull request. Need I do something rather than check?)
By the way, I'd like to translate your README.md into Chinese. Will it be OK for you?

**Comment by PeterlitsZo** [comment]
Add more commits by pushing to the master branch on PeterlitsZo/argparse.
I did. Now I think everything will be going well `^_^`. Let us merge it, can we?

**Comment by akamensky** [comment]
Merged

**Pull request merged**